packs/equipment/entertainers-cincture.json

```diff
@@ -73,16 +73,14 @@
                 "key": "ChoiceSet",
                 "predicate": [
                     {
-                        "not": [
-                            "item:uninvested"
-                        ]
+                        "not": "item:uninvested"
```

Comment:
Is this even a roll option?

Reply:
Sorry that's on me. I was testing stuff and forgot to remove it.